### ISSUE TITLE
Not able to add binary download in sources

### ISSUE THREAD

**Issue by ErikReider** [Elara6331/lure#79]
Lure won't download "https://github.com/zsa/wally/releases/download/2.1.3-linux/wally" for some reason. Is it because it's not a Git repo / archive?

**Comment by Elagoht** [comment]
I had the exact same problem. I fixed it by updating sources parameter as sources_amd64 because my application only supports amd64.

```sh
architectures=("amd64")
sources_amd64=("https://github.com/Elagoht/PomoDoIt/releases/download/v$version/pomodoit.tar.gz")
checksums_amd64=("f054c3a31c43df9fd7df003ddca7c859f12b0cc1067f1030a1adfc7012e4878f")
```

**Comment by ErikReider** [comment]
> I had the exact same problem. I fixed it by updating sources parameter as sources_amd64 because my application only supports amd64.
> 
> ```shell
> architectures=("amd64")
> sources_amd64=("https://github.com/Elagoht/PomoDoIt/releases/download/v$version/pomodoit.tar.gz")
> checksums_amd64=("f054c3a31c43df9fd7df003ddca7c859f12b0cc1067f1030a1adfc7012e4878f")
> ```

You're downloading a archive though, I'm trying to download an executable

**Comment by Elara6331** [comment]
Can you try with the latest commit? f44a9509a2bd121104afc1412b0a35488128b26f should fix it.

**Comment by ErikReider** [comment]
It downloads but the name gets all messed up

**Comment by ErikReider** [comment]
Adding `?~name=wally` to the end of the query worked as a workaround. I'll let you close this in case you think that this is a bug or not.

Thanks for your time! :)

**Comment by Elara6331** [comment]
Fixed in b758eb3
